NuGet validation fails with 1.6 references due to unexpected included webview2loader.dll

Got a report from building with 1.6-stable against our latest 1.6-preview2 bits with this error:

1>C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5321,5): error MSB3030: Could not copy the file "C:\Users....nuget\packages\communitytoolkit.winui.animations\8.2.240909-build.1181\lib\net8.0-windows10.0.22621\CommunityToolkit.WinUI.Animations\runtimes\win-x64\native\WebView2Loader.dll" because it was not found.

looks like a path problem:
 
Wrong
C:\Users....nuget\packages\communitytoolkit.winui.animations\8.2.240909-build.1181\lib\net8.0-windows10.0.22621\CommunityToolkit.WinUI.Animations\runtimes\win-x64\native\
 
Correct
C:\Users....nuget\packages\communitytoolkit.winui.animations\8.2.240909-build.1181\lib\net8.0-windows10.0.22621.0\CommunityToolkit.WinUI.Animations\runtimes\win-x64\native
Note the missing `.0` in the TFM part of the path, should be `net8.0-windows10.0.22621.0` but was `net8.0-windows10.0.22621`.

Talking with @Arlodotexe it sounds similar (though different) to the issue Morten had reported in Labs here: https://github.com/CommunityToolkit/Labs-Windows/pull/569

Though with all the WebView changes between preview1 and stable, we may just want to have them reproduce once we have a 1.6 build to try. Still an issue in 1.6 stable, seems like a regression in the platform.

_Originally posted by @michael-hawker in https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/205#issuecomment-2339279747_ - [Internal Platform Tracking Issue](https://microsoft.visualstudio.com/OS/_workitems/edit/53865998)

--- a/MultiTarget/PackageReferences/WinAppSdk.props
+++ b/MultiTarget/PackageReferences/WinAppSdk.props
@@ -2,7 +2,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240829007" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
-    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2792.45" />
+    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2792.45" PrivateAssets="all" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.1.1" />

--- a/MultiTarget/PackageReferences/WinAppSdk.props
+++ b/MultiTarget/PackageReferences/WinAppSdk.props
@@ -2,6 +2,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240829007" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
+    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2792.45" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.1.1" />

--- a/MultiTarget/WinUI.Extra.props
+++ b/MultiTarget/WinUI.Extra.props
@@ -36,6 +36,9 @@
 
     <WindowsSdkPackageVersion Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) >= 8">10.0.22621.41</WindowsSdkPackageVersion>
     <WindowsSdkPackageVersion Condition="8 > $([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)'))">10.0.22621.38</WindowsSdkPackageVersion>
+
+    <!--- Workaround for ADO 53865998 - See https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/215 - Don't include extraneous WebView2 dll -->
+    <WebView2NeverCopyLoaderDllToOutputDirectory Condition="'$(WebView2NeverCopyLoaderDllToOutputDirectory)' == '' And '$(WebView2EnableCsWinRTProjection)' == 'true'">true</WebView2NeverCopyLoaderDllToOutputDirectory>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(IsUno)' == 'true'">

--- a/MultiTarget/WinUI.Extra.props
+++ b/MultiTarget/WinUI.Extra.props
@@ -38,7 +38,7 @@
     <WindowsSdkPackageVersion Condition="8 > $([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)'))">10.0.22621.38</WindowsSdkPackageVersion>
 
     <!--- Workaround for ADO 53865998 - See https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/215 - Don't include extraneous WebView2 dll -->
-    <WebView2NeverCopyLoaderDllToOutputDirectory Condition="'$(WebView2NeverCopyLoaderDllToOutputDirectory)' == '' And '$(WebView2EnableCsWinRTProjection)' == 'true'">true</WebView2NeverCopyLoaderDllToOutputDirectory>
+    <WebView2NeverCopyLoaderDllToOutputDirectory>true</WebView2NeverCopyLoaderDllToOutputDirectory>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(IsUno)' == 'true'">

